.github/workflows/test-package.yml

```
@@ -47,7 +47,7 @@ jobs:
       matrix:
         # Add macos-latest and/or windows-latest if relevant for this package.
         os: [ubuntu-latest]
-        sdk: [dev]
+        sdk: [2.12.0, dev]
     steps:
       - uses: actions/checkout@v2
       - uses: dart-lang/setup-dart@v0.3
```

```suggestion
      - uses: dart-lang/setup-dart@v1.0
```

and other places here, too?

---

Done.